Tasks/Lysiura/Task5/task5.py

```diff
@@ -0,0 +1,87 @@
+def check_str(value):
+    l = 0
+    u= 0
+    for i in value:
+        if i.islower():
+            l = l+1
+        elif i.isupper():
+            u=u+1
+        else:
+            pass
+    print(f"{u} upper case, {l} lower case ")
+
+check_str("Hello Mikita")
+
+
+def is_prime(num):
+    if num in [1,2,3,5,7]:
+        return True
+    elif num%2 == 0 or num%2 == 0 or num%3 == 0 or num%7 == 0: 
+        return False
+    else:
+        return True
+
+print(is_prime(1020))
+print(is_prime(1019))
+
+def get_ranges(list_num):
+    try:
+       
+        mass1 = []
+        mass2 = [[],[],[],[],[],[],[],[],[],[],]
```

Comment:
what the hell is this abomination???

I'm not sure why do you need such declaration and hope that this is just a misplaced experiment